src/installer/test/Microsoft.NET.HostModel.Tests/Microsoft.NET.HostModel.Bundle.Tests/BundleExtractRun.cs

```diff
@@ -101,6 +101,18 @@ public SharedTestState()
                 RepoDirectories = new RepoDirectoriesProvider();
 
                 TestFixture = new TestProjectFixture("AppWithSubDirs", RepoDirectories);
+
+                // One of the sub-directories with a really long name is generated during test-runs
+                // rather than being checked in as a test asset.
+                // This prevents git-clone of the repo from failing if long-file-name support is not enabled on windows.
+                var longDirName = "This is a really, really, really, really, really, really, really, really, really, really, really, really, really, really long file name for punctuation";
+                var longDirPath = Path.Combine(TestFixture.TestProject.ProjectDirectory, "Sentence", longDirName);
+                Directory.CreateDirectory(longDirPath);
```

Comment:
Delete the directory when the test is done?

Reply:
The file is created in the TestArtifact location; the entire containing directory is disposed at the end of the test exectuion.